inline.md

```diff
@@ -0,0 +1,106 @@
+## paymentspring checkout form
+
+The checkout form is a simple way to seamlessly integrate the checkout button's ease of use natively into your site.
+
+The form elements are placed along side your html making the form part of your user experience and ultimately gives you the control.
+
+Each form is configurable with all the options found on our [button examples page](https://paymentspring.github.io/checkout-examples/).
+
+### Example
+
+To include a form on your website, you will need to obtain a form-id. Then copy and paste a line of javascript into your site where you would like the form to appear.
+
+```markdown
+<script src="https://checkout.paymentspring.com/js/inline.js" formId="cb7eae32763131ade35f"></script>
+```
+Where `cb7eae32763131ade35f` is your form-id.
+
+In addition to the javascript above, you will need to paste a link to our stylesheet (css) in the head of your site. If you plan to style the form, see styling below, please place this link before other css links.
+
+```markdown
+<link rel="stylesheet" type="text/css" href="https://checkout.paymentspring.com/assets/application.css">
+```
+
+{% include inline-checkout/inline-script.html %}
+
+### Styling
+
+One of the cool things about using the inline checkout form is that it is completely customizable to fit your site. By including the link to our stylesheet before your own you can easily override css rules to add colors, change sizes, and make our form your own.
+
+Here is an example that updates the inline checkout form to match the colors of the page.
+
+{% include inline-checkout/inline-example-image.html %}
+
+The sass (compiled to css) to accomplish the above looks like this.
+
+```
```

Comment:
Would it be worth also including the raw CSS that this compiles to, for folks who haven't ever used Sass or who don't have access to it for whatever reason?

Reply:
how difficult would it be to each of these in tabs? similar to http://sass-lang.com/guide#topic-2-Sass.